CONTRIBUTING.md

```diff
@@ -48,13 +48,13 @@ Codespaces is new technology that allows you to use a container as your developm
 ### GitHub Codespaces
 
 1. From the Azure SDK GitHub repo, click on the "Code -> Open with Codespaces" button.
-1. Open a Terminal. The development environment will be ready for you. Continue to [Building and Testing](#building-and-Testing).
```

Comment:
@sima-zhu are warnings like these intended to fail the build? If so, why is it a warning and not an error?

https://dev.azure.com/azure-sdk/public/_build/results?buildId=796379&view=logs&j=a129effc-2dd1-54d1-fb5a-ad7bdc0e851d&t=617ceecd-bf48-5ce3-83e2-7468b1632a13

```text
##[warning]Please lower case your anchor tags (i.e. anything after '#' in your link '#building-and-Testing'. Check here for more information: https://aka.ms/azsdk/guideline/links 
Found 27 links on page file:///D:/a/_work/1/s/README.md 
Found 12 links on page file:///D:/a/_work/1/s/SECURITY.md 
Summary of broken links: 
'file:///D:/a/_work/1/s/CONTRIBUTING.md' has 1 broken link(s): 
 #building-and-Testing
```